electron/main/llm/ipcHandlers.ts

```diff
@@ -38,4 +38,32 @@ export const registerLLMSessionHandlers = (store: Store<StoreSchema>) => {
     }
     await ollamaService.pullModel(modelName, handleProgress)
   })
+
+  ipcMain.handle('get-available-models', async () => {
```

Comment:
i dont think we need this ipc handler at all

Reply:
resolved!